guides/common/modules/proc_adding-custom-deb-repositories.adoc

```diff
@@ -0,0 +1,72 @@
+[id="Adding_Custom_DEB_Repositories_{context}"]
+= Adding {customdebtitle} Repositories
+
+Use this procedure to add {customdeb} repositories in {Project}.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-custom-deb-repositories[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products* and select the product that you want to use, and then click *New Repository*.
+. In the *Name* field, enter a name for the repository.
+{Project} automatically completes the *Label* field based on what you have entered for *Name*.
+. Optional: In the *Description* field, enter a description for the repository.
+. From the *Type* list, select `deb` as type of repository.
+. In the *Upstream URL* field, enter the URL of the external repository to use as a source.
+You can find the upstream URLs on Debian-based systems in `/etc/apt/sources.list`.
+. Optional: In the *Releases* field, set a or multiple releases.
++
+For official Debian repositories, {Team} recommends setting a _codename_ in the *Releases* field, for example `buster` for Debian 10 or `bullseye` for Debian 11.
+Do not use `stable` or `testing` as they merely reference a currently corresponding _codename_.
+This helps to avoid drastic changes once a new Debian version is released and the reference is changed.
+{Team} recommends creating one repository in {Project} per release to keep things tidy and avoid potential performance and network issues during synchronization.
+For official Ubuntu repositories, use the Ubuntu suite, for example `focal` or `focal-updates`.
+. Optional: In the *Components* field, enter a component.
+This indicates the licensing terms of the software packages.
++
+In Debian, it is divided into `main`, `contrib`, and `non-free`.
+For official Debian or Ubuntu repositories, {Team} recommends leaving this field empty to synchronize all available components.
+Note that some third party Debian repositories use the components in ways that may require setting an explicit selection.
++
+[CAUTION]
+====
+Ensure to enter both *Releases* and *Components* exactly as they would be in an `/etc/apt/sources.list` file.
```

Comment:
This sounds quite strange to be honest.
```suggestion
Ensure that you enter both *Releases* and *Components* exactly as they would be in an `/etc/apt/sources.list` file.
```
What do you mean by would be? We generally avoid would/should/could.

Reply:
Applied and reworded ~would be~ to are.